src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BuilderUtils.java

```diff
@@ -6,35 +6,94 @@
 
 package com.ca.apim.gateway.cagatewayconfig.util.gateway;
 
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle.loader.DependencyBundleLoadException;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=62&issues=AWVkDSPW4Bfu85TPS_PJ&open=AWVkDSPW4Bfu85TPS_PJ)

[ISSUE_KEY]: AWVkDSPW4Bfu85TPS_PJ